flutter/lib/benchmark/benchmark.dart

```diff
@@ -307,6 +307,11 @@ class BenchmarkState extends ChangeNotifier {
               break;
             }
           }
+        } else if (Platform.isAndroid) {
+          batchPreset = BatchPreset(
```

Comment:
Can you make `shardsCount` a constant and maybe add a brief comment why is it exactly 2?

Reply:
Done